systest/scripts/run_agent_transfers.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
I vote to rename this script to something that captures the spirit of the tests better, but not now. Perhaps next time we add tests.